src/tslint-cli.ts

```diff
@@ -207,6 +207,22 @@ const processFile = (file: string) => {
         process.exit(1);
     }
 
+    const buffer = new Buffer(256);
+    buffer.fill(0);
+    const fd = fs.openSync(file, "r");
+    try {
+        fs.readSync(fd, buffer, 0, 256, null);
+        if (buffer.readInt8(0) === 0x47 && buffer.readInt8(188) === 0x47) {
+            // MPEG transport streams use the '.ts' file extension. They use 0x47 as the frame
+            // separator, repeating every 188 bytes. It is unlikely to find that pattern in
```

Comment:
One last concern, sorry I didn't notice this before. `0x47` is "G", which means a TS file where the first and the 189th character are "G" won't be linted. Admittedly, it's not highly likely that this condition is met, but it seems like a non-trivial likelihood, no?